Packs/Core/Playbooks/playbook-External_Login_Password_Spray.yml

```diff
@@ -0,0 +1,1474 @@
+id: External Login Password Spray
+version: -1
+name: External Login Password Spray
+description: |-
+  This playbook investigates and responds to external login password sprays. It starts by enriching the external IP, allowing early containment. It then fetches event information to learn more about the attack - how it was executed and whether it succeeded or not.
```

Comment:
This playbook investigates and responds to external login password sprays. It starts by enriching the external IP, allowing early containment. It then fetches event information to learn more about the attack - how it was executed and whether it succeeded or notThis playbook is designed to investigate and respond to external login password sprays. It begins by enriching the external IP to enable early containment. Then, it retrieves event information to gain insights into the attack, including how it was carried out and whether it was successful.